web_app/calendar/app.py

```diff
@@ -82,6 +105,8 @@ def get_days_page_for_month(year_id, month_id):
 @app.route(hours_route)
 @app.route(hours_route + '/')
 def get_hours_page_for_day(year_id, month_id, day_id):
+    base_url = request.base_url
```

Comment:
Why do you use request.base_url here? Does it differ from hours_route somehow?

Reply:
Yeap, hours_route is template like  - /years/<year_id>/... but  request.base_url - 
it's url alive with set parameters